translate/src/modules/entitieslist/components/EntitiesList.test.js

```diff
@@ -1,12 +1,47 @@
+/* global global */
+
 import { createMemoryHistory } from 'history';
+import { act } from 'react-dom/test-utils';
 import sinon from 'sinon';
 
 import * as EntitiesActions from '~/core/entities/actions';
 import * as BatchActions from '~/modules/batchactions/actions';
-import { createReduxStore, mountComponentWithStore } from '~/test/store';
+
+import {
+  createDefaultUser,
+  createReduxStore,
+  mountComponentWithStore,
+} from '~/test/store';
 
 import EntitiesList, { EntitiesListBase } from './EntitiesList';
```

Comment:
`EntitiesListBase` is never used.

Reply:
Refactored the exports so EntitiesListBase no longer exists.